keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -1258,6 +1281,31 @@ more confusing for users. Having just one simple deciding factor based on
 ServiceExport oldness makes resolving conflicts straightforward, and this
 alternative conflict resolution algorithm could hinder this ease of use.
 
+### Exporting labels/annotations from the Service/ServiceExport objects
+
+`Service` and `ServiceExport` have labels and annotations which could be used during
+export and propagated to the `ServiceImport`. However various tools such as kubectl or
+ArgoCD add some labels and annotations which would then need to be actively
+filtered to avoid any conflict. Filtering those labels and annotations is not
+something easy and we chose to avoid this problem entirely by not using the metadata
+object and adding dedicated fields in the spec of the `ServiceExport` resource.
+
+Also if we were using the labels and annotations from the metadata of either the
+`ServiceExport` or `Service` resources, it may be more confusing for users as it
+would be the only fields present in both resources. For instance, should an
+implementation merge the labels/annotations from both objects? Should it favor one?
+Should it takes only from the `Service` object? With dedicated fields for labels
+and annotations in the spec of the `ServiceExport` resource, it may becomes more
+straightforward that each resource have their own labels and annotations in their
+metadata and that the exported labels and annotations are from the dedicated
+fields in the `ServiceExport` spec.
+
+We also favored dedicated fields on the `ServiceExport` resource to allow for better
+flexibility, as it will allow to export labels and annotations fully decorrelated
+from the `Service` and `ServiceExport` metadata. More flexibility could also be
+achieved with CEL expression on the `ServiceExport` at the cost of greater
```

Comment:
```suggestion
from the `Service` and `ServiceExport` metadata. 

This implementation does not preclude the possibility of additionally adopting a
"copy/filter" strategy (either as global configuration for a particular implementation, or
granularly as additional `ServiceExport` fields) if the need arises. Such an strategy could
select specific label or annotation keys to sync from the `Service` by exact name, by
prefix, or further flexibility could be achieved with CEL expressions at the cost of
greater
```

Feel free to reject this suggestion - from the long thread above this felt like this possibility could be worth mentioning but perhaps the "`exportedLabels` and `exportedAnnotations` MUST take precedence" snippet I suggested adding above is a sufficent oblique reference without introducing confusion.